packages/api/src/app-router.ts

```diff
@@ -32,6 +33,7 @@ export const appRouter = t.router({
     .input(z.void())
     .output(z.string())
     .query(() => "yay!"),
+  getRollups,
```

Comment:
These changes were removed in the rollup filter refactor. This should be an error in managing conflicts after many merges